vizro-core/src/vizro/_vizro.py

```diff
@@ -51,7 +48,6 @@ def build(self, dashboard: Dashboard):
         self._pre_build()
 
         self.dash.layout = dashboard.build()
-        _build_app_callbacks()
```

Comment:
When is this now done?

Reply:
Inside the Dashboard.build method. Line: `children=[*ActionManager._create_app_callbacks(), dash.page_container],`.

`ActionManager._create_app_callbacks()` builds all action callbacks and returns all action components.